lib/metric-config-parser/metric_config_parser/tests/test_experiment.py

```diff
@@ -376,6 +378,30 @@ def test_sample_size_none_experiment(self, experiments, config_collection):
         cfg = spec.resolve(experiments[7], config_collection)
         assert cfg.experiment.sample_size is None
 
+    def test_analysis_unit_default(self, experiments, config_collection):
+        conf = dedent(
+            """
+            [experiment]
+            enrollment_period = 7
+            """
+        )
+        spec = AnalysisSpec.from_dict(toml.loads(conf))
+        cfg = spec.resolve(experiments[1], config_collection)
+        assert cfg.experiment.randomization_unit is None
+        assert cfg.experiment.analysis_unit == AnalysisUnit.CLIENT
+
+    def test_analysis_unit_configured(self, experiments, config_collection):
+        conf = dedent(
+            """
+            [experiment]
+            enrollment_period = 7
+            """
+        )
+        spec = AnalysisSpec.from_dict(toml.loads(conf))
+        cfg = spec.resolve(experiments[0], config_collection)
```

Comment:
`experiments` is the list of `Experiment`s defined in `conftest.py`?

Reply:
Yep, correct.